arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -142,7 +142,7 @@ public async Task<PostDocumentResponse<T>> PutDocumentAsync<T>(string documentId
         /// <returns></returns>
         public async Task<T> GetDocumentAsync<T>(string collectionName, string documentKey)
         {
-            return await GetDocumentAsync<T>($"{collectionName}/{documentKey}");
+            return await GetDocumentAsync<T>($"{WebUtility.UrlEncode(collectionName)}/{documentKey}");
```

Comment:
We also need to URL encode `documentKey`

Reply:
My assumption here was that the documentId should not be user defined?